### ISSUE TITLE
esp to close a dialog no longer works if you click some text in the dialog

### ISSUE THREAD

**Issue by dbkr** [vector-im/riot-web#6623]

**Comment by lampholder** [comment]
I think it's actually when the textbox doesn't have focus rather than clicking text.

/me updates the issue title

I think you have to give HTML elements a tabindex property if you want them to receive keypress events? Anyway there might be a less grim way of doing that if you actually know what you're doing :P

**Comment by dbkr** [comment]
Yeah, must be something to do with what element has the focus. Ftr, exact repro steps for this are to click, 'Submit debug logs', then click anything in the dialog that isn't a form control, then press escape (the dialog won't close).

**Comment by aaronraimist** [comment]
Hmm if you click somewhere on the dialog you have to press escape twice to close it, the first escape doesn't do anything

**Comment by t3chguy** [comment]
I believe this is related to focus-trap, will put it on my backlog

**Comment by t3chguy** [comment]
Does not look to be focus-trap related

**Comment by t3chguy** [comment]
Upstream https://github.com/theKashey/react-focus-lock/issues/129
